modules/monitoring-creating-alerting-rules-for-user-defined-projects.adoc

```diff
@@ -8,6 +8,13 @@
 
 You can create alerting rules for user-defined projects. Those alerting rules will fire alerts based on the values of chosen metrics.
 
+[WARNING]
+====
+If you deploy an alerting rule directly on the Prometheus instance in the `openshift-user-workload-monitoring` project by adding the `leaf-prometheus` label to the rule, only those metrics exposed by your user-defined project will be available to use in the alerting rule.
```

Comment:
`leaf-prometheus` is not label, label with the key `openshift.io/prometheus-rule-evaluation-scope` and value `leaf-prometheus`

Reply:
@juzhao - Updated per your suggestion. PTAL.